app/controllers/application_controller.rb

```diff
@@ -27,7 +33,7 @@ def namespace
 
   def user_not_authorized(_exception)
     flash[:alert] = t('flash.not_authorized')
-    redirect_to(request.referer || root_path)
+    redirect_to(page_path('subscribe') || root_path)
```

Comment:
This page should be on `SubscriptionsController#new` instead of in a static page, don't you think?

Reply:
I'd like to talk about the step-by-step sign-up/subscription flow to get this business logic clear.